cmd/operator-sdk/test/local.go

```diff
@@ -293,6 +243,162 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func collectManifests() error {
+	// Pass empty files to the test-framework if no setup required
+	if tlConfig.noSetup {
+		emptyBytes := []byte{}
+		file, err := writeToTempFile(emptyBytes, "empty.yaml")
+		if err != nil {
+			return err
+		}
+		tlConfig.namespacedManPath = file.Name()
+		tlConfig.globalManPath = file.Name()
+
+		f := func() {
+			err := os.Remove(file.Name())
+			if err != nil {
+				log.Errorf("Could not delete empty manifest file: (%v)", err)
+			}
+		}
+		cleanups = append(cleanups, f)
+		return nil
+	}
```

Comment:
WDYT about encapsulated the logic to be more easier get what it does? E.g

```suggestion
	if tlConfig.noSetup {
		if err := tlConfig.generateDefaultSetup(); err!=nil{
		    return err
		}
	}
```